chromatic/rainbows/actions/conversions.py

```diff
@@ -0,0 +1,79 @@
+from ...imports import *
+
+__all__ = ["to_nparray", "to_df"]
+
+
+def to_nparray(self, timeformat='h'):
+    """ Convert Rainbow object to 1D and 2D numpy arrays
+    Parameters
+    ----------
+        self : Rainbow object
+            chromatic Rainbow object to convert into array format
+        timeformat : str
+            (optional, default='hours')
+            The time format to use (seconds, minutes, hours, days etc.)
+    Returns
+    ----------
+        rflux : np.array
+            flux (MJy/sr)        [n_wavelengths x n_integrations]
```

Comment:
Here we shouldn't always assume that flux will be in units of MJy/sr; I think that is really just kind of a weird specific thing coming out of the `jwst` pipeline. We'll often have unitless fluxes, or photons, or electrons/s, or W/m^2/nm, or whatever other weird unit. Could we chance the docstring to be agnostic about the units of this flux?

Reply:
Done!